src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/BlogPostsController.java

```diff
@@ -0,0 +1,30 @@
+package pl.tomaszdziurko.jvm_bloggers.blog_posts;
+
+import java.io.PrintWriter;
+
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.sun.syndication.io.SyndFeedOutput;
+
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+
+@RestController
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class BlogPostsController {
+
+    private final AggregatedRssFeedProducer rssProducer;
+
+    @SneakyThrows
```

Comment:
Hmm...I do not think that "masking" exceptions is a good idea on REST endpoints;/ I am just thinking what Swagger2 will say when we decide to use it in the future? Can we just use @ControllerAdvice and handle exception?